### ISSUE TITLE
xgboost import error in windows python

### ISSUE THREAD

**Issue by zy1369834** [dmlc/xgboost#791]
Hi,

I have installed xgboost in my  python 2.7.11 (windows 8.1, 64bit) by following guide from github. This procedure is smoothly and successful. But when importing xgboost, os.winerror 193 appears. The detail is below:
![image](https://cloud.githubusercontent.com/assets/7645535/12697545/1310344a-c787-11e5-92f2-bf6919a61570.png)

i also tried other python version, still not work. Hope you can help me to solve it.

Thanks a lot

Yi

**Comment by thushv89** [comment]
Hi,

I got the same error because I ran "make" after removing the flag "-m64" from the makefile (config.mk) check whether you have that in the makefile

**Comment by zy1369834** [comment]
My config.mk file has '-m64'. No matter removing that or not, the same error occurs when try to import. But I notice some warnings during setup. Just wander whether it s the reason.
![image](https://cloud.githubusercontent.com/assets/7645535/12698784/801d88f2-c7a6-11e5-92f2-d79789823806.png)

**Comment by tqchen** [comment]
If you are using 32 bit python, try to remove -m32, remember to type 
```
make clean_all;
```
before you build again

**Comment by thushv89** [comment]
Clearly this is a 32-bit/64-bit incompatibility. I got around this with visual studio (but for a version 0.4)
If the above doesn't work. See if this helps. I couldn't get it to work with any of the make tools.
[kaggle post](https://www.kaggle.com/c/otto-group-product-classification-challenge/forums/t/13043/run-xgboost-from-windows-and-python)

also, if that's not clear, I wrote a more detailed blogpost inspired from the above one.
[blog post](http://www.thushv.com/uncategorized/i-knew-you-were-trouble-xgboost-windows-a-proper-guide/)
